src/order-actions/reject-order.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useCallback } from "react";
+import { OverflowMenuItem, OverflowMenu } from "@carbon/react";
+import { useTranslation } from "react-i18next";
+import { showModal } from "@openmrs/esm-framework";
+import { Result } from "../work-list/work-list.resource";
+
+interface RejectOrderOverflowMenuItemProps {
+  order: Result;
+}
+
+const RejectOrderOverflowMenuItem: React.FC<
```

Comment:
Well done @gitcliff  Instead of using  `const { t } = useTranslation();` directly, You can also destructure 't ' directly in the function. 
i guess it might look like

 ```
const RejectOrderOverflowMenuItem: React.FC<RejectOrderOverflowMenuItemProps> = ({ order }) => {
  const { t } = useTranslation();
```